website/docs/r/pi_virtual_serial_number.html.markdown

```diff
@@ -0,0 +1,74 @@
+---
+
+subcategory: "Power Systems"
+layout: "ibm"
+page_title: "IBM: pi_virtual_serial_number"
+description: |-
+  Manages an EXISTING virtual serial number in IBM Power
+---
+
+# ibm_pi_virtual_serial_number
+Get, update or delete an existing virtual serial number.
+
+## Example usage
+The following example enables you to create a virtual serial number:
+
+```terraform
+resource "ibm_pi_virtual_serial_number" "testacc_virtual_serial_number" {
+  pi_serial            = "<existing virtual serial number>"
+  pi_cloud_instance_id = "<value of the cloud_instance_id>"
+  pi_description       = "<desired description for virtual serial number>"
+}
+```
+
+**Note**
+* Please find [supported Regions](https://cloud.ibm.com/apidocs/power-cloud#endpoint) for endpoints.
```

Comment:
Replace stars with dashes.

```suggestion
- Please find [supported Regions](https://cloud.ibm.com/apidocs/power-cloud#endpoint) for endpoints.
```

This wasn't flagged as linting error by the markdown linter?

Reply:
Fixed